pkg/JuliaInterface/gap/utils.gi

```diff
@@ -4,6 +4,16 @@
 ##
 #############################################################################
 
+BindGlobal( "JuliaSetVal",
+function( name, val )
+  if not IsString( name ) then
+    Error( "JuliaSetVal: <name> must be a string (not the value '", name, "')" );
+  else
+    JuliaFunction( "_setglobal!", "GAP" )( Julia.Main,
+        Julia.Symbol( GAPToJulia( name ) ), val );
```

Comment:
This also works, as we've installed a `Symbol` constructor for GAP strings:
```suggestion
    JuliaFunction( "_setglobal!", "GAP" )( Julia.Main,
        Julia.Symbol( name ), val );
```

We also have a dedicated function for this, which is faster; not that it matters much here, it is mostly for `_JL_RNAM_TO_JULIA_SYMBOL`.
```suggestion
    JuliaFunction( "_setglobal!", "GAP" )( Julia.Main,
        JuliaSymbol( name ), val );
```